packages/ast-utilities/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+### Added
+
+- `replaceJsxBody()` support root nodes
```

Comment:
See https://github.com/Shopify/quilt/pull/2085, will do the minor changelog clean up before merging.